src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends AbstractFigure {
+    private double base1;
+    private double base2;
+    private double heigth;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.heigth = height;
+    }
+
+    public double getArea() {
+        return 0.5 * (base1 + base2) * heigth;
+    }
+
+    @Override
+    public String describe() {
+        return "IsoscelesTrapezoid " + getColor() + " base1: " + base1 + " base2: " + base2
+                + " height: " + heigth + " area: " + getArea();
```

Comment:
The variable `heigth` is misspelled. It should be `height` to maintain code clarity and prevent potential bugs.